firefox-ios/Client/Coordinators/Router/Route.swift

```diff
@@ -65,6 +65,7 @@ enum Route: Equatable {
     /// - Parameter section: An instance of `DefaultBrowserSection` indicating the section of the default browser
     ///                      settings to be displayed.
     case defaultBrowser(section: DefaultBrowserSection)
+    case sharesheet(url: URL, title: String?)
```

Comment:
we should add documentation for this, like the rest of the functions defined here.